src/systems/__snapshots__/systemItemsTable.component.test.tsx.snap

```diff
@@ -1873,7 +2183,7 @@ exports[`SystemItemsTable > SystemItemsTable (normal) > renders correctly 1`] =
               data-index="6"
             >
               <div
-                aria-label="Tue Sep 05 2023 23:00:00 GMT+0000 (Coordinated Universal Time),Tue Sep 05 2023 23:00:00 GMT+0000 (Coordinated Universal Time)"
+                aria-label=",Tue Sep 05 2023 23:00:00 GMT+0000 (Coordinated Universal Time)"
```

Comment:
```suggestion
                aria-label="Tue Sep 05 2023 23:00:00 GMT+0000 (Coordinated Universal Time)"
```

Should the comma be there?

Reply:
I don't expect it to be there, so I'm not sure why it does appear?  It could be a follow on from the previous dates in separate divs; the snapshot might split up dates listed.